autoconnect/autoconnect-web/src/routes.rs

```diff
@@ -43,9 +43,11 @@ pub async fn push_route(
             .await;
     }
     // Attempt to send the notification to the UA using WebSocket protocol, or store on failure.
+    // NOTE: Since this clones the notification, there is a potential to
+    // double count the reliability state.
     let result = app_state
         .clients
-        .notify(uaid.into_inner(), notif.clone())
+        .notify(uaid.into_inner(), notif.clone_without_reliability_state())
```

Comment:
This could be inlined, but I'll let the compiler do that. Having the function makes it descriptive.